src/generateTypeScriptTypes.ts

```diff
@@ -1,13 +1,12 @@
 #!/usr/bin/env node
-
-import { program } from "commander";
-import { version } from "../package.json";
+import GeneratorProgram from "./GeneratorProgram";
 import TypeScriptGenerator from "./generators/TypeScriptGenerator";
-import parser from "./parser";
+import { program } from "commander";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Import sources within a group must be alphabetized.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5610987)